Code of Conduct/Sociomantic Code of Conduct.md

```diff
@@ -0,0 +1,75 @@
+# Sociomantic Code of Conduct
+
+## Purpose
+
+As our company mission states, we are here to continuously innovate the world’s most trusted personalized media platform and that vital _trust_ is what bonds us not only to our clients, but also to each other. It’s our intention to help define and foster such trust with the use of a shared Code of Conduct. These commitments should represent our people and our company to provide us a safe work environment and reputable image.
+
+## Contents
+
+ * [Purpose](#purpose)
+ * [Contents](#contents)
+ * [Our Commitments](#our-commitments)
+    * [People at Sociomantic Labs](#1.-people-at-sociomantic-labs)
+    * [Ethical Work Products](#2.-ethical-work-products)
+    * [Sustainability and Social Responsibility](#3.-sustainability-and-social-responsibility)
+    * [Integrity towards Customers, Suppliers and Partners](4.-integrity-towards-customers,-suppliers-and-partners)
+    * [Conflicts of Interest](5.-conflicts-of-interest)
+    * [Competition](6.-competition)
+    * [Laws and Regulations](7.-laws-and-regulations)
+    * [Accurate Accounting](8.-accurate-accounting)
+    * [Prevention of Bribery and Corruption](9.-prevention-of-bribery-and-corruption)
+    * [Company Resources](10.-company-resources)
+    * [Confidential Information and Personal Data](11.-confidential-informationand-personal-data)
+    * [Corporate Identity and Reputation](12.-corporate-identity-and-reputation)
+    * [Politics](#13.-politics)
+ * [Speak Up](#speak-up)
+ * [Related Policies](#related-policies)
+ * [Contributions](#contributions)
+
+## Our Commitments
+
+### 1. People at Sociomantic Labs
+
+Sociomantic’s success depends on us, the people who form this company and contribute to its growth and evolution. As a global company, we embrace diversity, which gives Sociomantic its unique character. We want to create an environment of equality and inclusion where each of us is valued, well-treated and encouraged to reach their individual potential.
+
We have the right to fair, courteous and respectful treatment by our colleagues. Any kind of discrimination or harassment (such as intimidation, hostility, humiliation, degradation or insult) is strictly forbidden.
```

Comment:
Mmm, it looks like there are some weird end of lines, I guess they are CR characters from Windows/DOS-style end of line.